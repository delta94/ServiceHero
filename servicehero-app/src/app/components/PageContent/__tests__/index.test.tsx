import * as React from 'react';
import { render } from '@testing-library/react';

import { PageContent } from '..';

describe('<PageContent  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<PageContent children={() => <></>} />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
