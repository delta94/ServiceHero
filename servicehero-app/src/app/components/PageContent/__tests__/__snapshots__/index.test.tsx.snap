// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageContent  /> should match snapshot 1`] = `
.c0 {
  min-height: calc(100vh - 104px);
  padding: 30px 60px;
  position: relative;
}

<main
  class="ant-layout-content c0"
/>
`;
